hack/deploy_csv.yaml

```diff
@@ -311,6 +311,8 @@ data:
           * OpenShift Pipelines
           ## Installation
           CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in default `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-workspaces` namespace.
+          ##### Prerequisite:
+          * To finish the installation of Red Hat CodeReady Workspaces Operator, it is necessary that at least one real user exists in the OpenShift cluster.
```

Comment:
Not sure about the `real user` wording here... But let's Parag correct it if needed :)